content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md

```diff
@@ -54,31 +54,32 @@ The .NET Tracer supports automatic instrumentation on the following architecture
-| Kafka                           | `Confluent.Kafka` 1.4+                                                                    | `Kafka`              |
-| MongoDB                         | `MongoDB.Driver.Core` 2.1.0+                                                              | `MongoDb`            |
-| MySql                           | `MySql.Data` 6.7.0+</br>`MySqlConnector` 0.61.0+                                          | `MySql`              |
-| Oracle                          | `Oracle.ManagedDataAccess` 4.122.0+                                                       | `Oracle`             |
-| PostgreSQL                      | `Npgsql` 4.0+                                                                             | `Npgsql`             |
-| RabbitMQ                        | `RabbitMQ.Client` 3.6.9+                                                                  | `RabbitMQ`           |
-| Redis (ServiceStack client)     | `ServiceStack.Redis` 4.0.48+                                                              | `ServiceStackRedis`  |
-| Redis (StackExchange client)    | `StackExchange.Redis` 1.0.187+                                                            | `StackExchangeRedis` |
-| Service Fabric Remoting         | `Microsoft.ServiceFabric.Services.Remoting` 4.0.470+                                      | `ServiceRemoting`    |
-| SQLite                          | `System.Data.Sqlite` 2.0.0+ </br>`Microsoft.Data.Sqlite` 1.0.0+                           | `Sqlite`             |
-| SQL Server                      | `System.Data` 4.0.0+</br>`System.Data.SqlClient` 4.0.0+</br>`Microsoft.Data.SqlClient` 1.0.0+  | `SqlClient`     |
-| WCF (server)                    | built-in                                                                                  | `Wcf`                |
-| WebClient / WebRequest          | `System.Net.Requests` 4.0+                                                                | `WebRequest`         |
+| Framework or library            | NuGet package                                                                                        | Integration Name     |
+| ------------------------------- | ---------------------------------------------------------------------------------------------------- | -------------------- |
+| ADO.NET                         | All AdoNet integrations                                                                              | `AdoNet`             |
+| Aerospike                       | `Aerospike.Client` 4.0.0+                                                                            | `Aerospike`          |
+| ASP.NET Core                    | `Microsoft.AspNetCore`</br>`Microsoft.AspNetCore.App`</br>2.0+ and 3.0+                              | `AspNetCore`         |
+| Azure Functions                 | `Microsoft.Azure.Webjobs` 3.0+                                                                       | `AzureFunctions`     |
```

Comment:
@pierotibou @andrewlock, what do you think of including Azure Functions here? It's a little different than other integrations, so I'm not sure.

Reply:
LGTM, we do instrument this assembly, it generates a span. Makes sense to be here

Reply:
I wonder if it's worth pointing out _which_ AzureFunctions libraries we instrument though? e.g. which versions/inproc/outofproc etc (being vague as I can't remember 😂 )